macOS/DuckDuckGo/TabBar/View/TabBarCollectionView.swift

```diff
@@ -153,6 +153,11 @@ open class TabBarCollectionView: NSCollectionView {
         }
     }
 
+    func itemNext(to indexPath: IndexPath) -> NSCollectionViewItem? {
```

Comment:
Naming nit-pick: “item next to” could be interpreted as either previous or next item.

How about `nextItem(for indexPath: IndexPath)`?